handbook/engineering/distribution/goals.md

```diff
@@ -20,12 +20,12 @@ Our existing deployment pipelines to our Sourcegraph instances (such as Sourcegr
   - TBD: We can trigger rollbacks and deployments via a `/` command in Slack.
 - **Milestones**: TBD
 
-### Any engineer at Sourcegraph can create a release for all of our supported deployment types by running a single command
+### Any engineer at Sourcegraph can create a release for all of our supported deployment types by running a single action
```

Comment:
I think "a single command", and "slack command" at that, is not so important. I suspect that we would be OK with this being a single action of some sort (whether that is a CLI command, creating a GitHub issue, etc.) as long as the rest is automated.